servers/dsub/jobs/swagger/swagger.yaml

```diff
@@ -230,6 +230,9 @@ definitions:
       attempts:
         type: "integer"
         description: "Number of times the task was run"
+      jobId:
+        type: "string"
+        description: "Id of the job corresponding to this task"
```

Comment:
Could you make this a bit more explicit? Too easy to confuse with the parent's job ID right now. Maybe:

..., if this task corresponds to a nested child job. Expresses a child:parent relationship between the provided job and the job containing this task.

Reply:
Yeah it's confusing that the jobId corresponds to the child job/current task here, not the parent job